core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -134,13 +135,16 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging {
 
   /** Set multiple parameters together */
   def setAll(settings: Traversable[(String, String)]): SparkConf = {
+    settings.foreach { case (k, v) => logDeprecationWarning(k) }
```

Comment:
Not your change, but I wonder if this one should just keep calling `set`. This currently doesn't handle null keys or values